test/cli/test_cli_functions.sh

```diff
@@ -284,10 +283,97 @@ function obc_cycle {
     echo_time "✅  obc cycle is done"
 }
 
+function crd_cycle {  
+    crd_array=($(kubectl get crd | awk '{print $1}' | grep -v "NAME"))
```

Comment:
Why isn't it local?

Reply:
Also used in another function - uninstall with cleanup